### DISCUSSION TITLE
rpgle files read only

### DISCUSSION THREAD

**Discussion by uzandav** [codefori/vscode-ibmi#1491] [IBM i]
Good morning,
I have a problem with VS code, all my rpgle sources on ibmi have been read only for a few days. I can't modify them in VS code because I have the message "editor is read-only because the file is read-only", a padlock appears in front of each file.
![2023-08-19_14h54_39](https://github.com/halcyon-tech/vscode-ibmi/assets/100574138/4f7b28c4-bc50-42b5-b6c0-2665c7a5aa9d)

Thank you in advance for your help.
VS code 1.8.1
Extension Code for IBM i v2.0.2

**Comment by aguerra72** [comment]
I have a same problem!!
any solution?

**Comment by chrjorgensen** [comment]
@aguerra72 

This discussion is closed...

You should first check the [issues section](https://github.com/codefori/vscode-ibmi/issues) in [the project repository](https://github.com/codefori/vscode-ibmi) and then the [pull requests](https://github.com/codefori/vscode-ibmi/pulls), to see if your problem has been solved or a workaround/explanation is found.

If no luck in that, please open a new issue from the `HELP AND SUPPORT > Report an Issue` option - this will create a report in the right spot and include additional information helping in debugging your problem.
